WcaOnRails/spec/models/competition_spec.rb

```diff
@@ -115,9 +115,22 @@
     expect(competition.is_probably_over?).to be false
     expect(competition.started?).to be false
     expect(competition.in_progress?).to be false
+    expect(competition.has_date?).to be false
     expect(competition.dangerously_close_to?(competition2)).to be false
   end
 
+  it "calculates the correct days until another future competition" do
+    competition = FactoryBot.build :competition, start_date: Date.parse("2021-01-01"), end_date: Date.parse("2021-01-03")
+    competition2 = FactoryBot.build :competition, start_date: Date.parse("2021-02-01"), end_date: Date.parse("2021-02-02")
+    expect(competition.days_until_competition?(competition2)).to be 29
+  end
+
+  it "calculates the correct days until another past competition" do
+    competition = FactoryBot.build :competition, start_date: Date.parse("2021-02-01"), end_date: Date.parse("2021-02-02")
+    competition2 = FactoryBot.build :competition, start_date: Date.parse("2021-01-01"), end_date: Date.parse("2021-01-03")
+    expect(competition.days_until_competition?(competition2)).to be (-29)
```

Comment:
Not to sure what to do about this
`to be -29` RuboCop gave the issue `/home/travis/build/thewca/worldcubeassociation.org/WcaOnRails/spec/models/competition_spec.rb:131:69: W: Lint/AmbiguousOperator: Ambiguous negative number operator. Parenthesize the method arguments if it's surely a negative number operator, or add a whitespace to the right of the `-` if it should be a subtraction.`

and with  `to be (-29)` gave the issue `/home/travis/build/thewca/worldcubeassociation.org/WcaOnRails/spec/models/competition_spec.rb:131:68: W: Lint/ParenthesesAsGroupedExpression: `(...)` interpreted as grouped expression.`

Reply:
Did you try `to be(-29)`?

Reply:
That did indeed work! Ill squash the commits since its kinda messy